05/app.js

```diff
@@ -9,6 +9,34 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const pElement = document.querySelectorAll('p');
+const counter = function(e) {
+    e.preventDefault();
+    const link = e.target.getAttribute("href");
+    const id = e.target.dataset.id;
+
+    if(link === "/dolor.html") {
```

Comment:
To podejście nie bardzo złe. Jeśli będzie 1000 linków to będzie 1000 if-ów.
Postaraj się pobrać dane do zmiennej (to już masz) np. `link` i teraz wykorzystaj.

np. `stats.links[link]++` tylko musisz sprawdzać czy już takie element istnieje jak nie to najpierw `stats.links[link]++` analogicznie powinieneś postąpić z paragrafem.